src/Containers/Article/Create/createArticle.jsx

```diff
@@ -0,0 +1,255 @@
+import React, { Component } from 'react';
+import { PropTypes } from 'prop-types';
+import { connect } from 'react-redux';
+import { ImageSideButton, Block, addNewBlock, createEditorState, Editor } from 'medium-draft';
+import axios from 'axios';
+import LoadingBar from 'react-top-loading-bar';
+import { ToastContainer } from 'react-toastify';
+import mediumDraftExporter from 'medium-draft/lib/exporter';
+import { faFeather } from '@fortawesome/free-solid-svg-icons';
+import publishArticle from '../../../actions/article';
+import getUser from '../../../actions/userInfo';
+import Sidebar from '../../../Components/SideBar/SideBar';
+import Navbar from '../../../Components/NavBar/NavBar';
+import Button from '../../../Components/Common/Button/Button';
+import './createArticle.scss';
+import 'react-toastify/dist/ReactToastify.css';
+
+let images = [];
+let spinner = false;
+
+/**
+ *
+ * @param {object} editorState
+ * @returns {method} render
+ */
+export class CustomImageSideButton extends ImageSideButton {
```

Comment:
Can you find a way to modularize this file by moving this class into its own file?